### ISSUE TITLE
[feature request] Disable autoplay while autotracking

### ISSUE THREAD

**Issue by nickkolok** [OpenSourcePhysics/tracker#19]
An option to disable autoplay (or to play with step **n**) will make autotracker faster.

**Comment by dobrown** [comment]
Good idea, I'll check it out. It could take a while, though, since I'm retired ;-) 

What kind of coding do you do? Do you know a lot about video and Java?

**Comment by dobrown** [comment]
Does the autotracker run any faster when the video is not visible (uncheck Video|Visible box)?

**Comment by nickkolok** [comment]
> What kind of coding do you do? Do you know a lot about video and Java?
My primary language is JavaScript. *Java relates to JavaScript as a dog does to a hot-dog.* (Russian version of the joke: *Java для JavaScript --  как кот для котлеты*). You can check my github if you want. So, I can probably help with debug. I'm not familiar to Java build system.

**Comment by nickkolok** [comment]
> Does the autotracker run any faster when the video is not visible (uncheck Video|Visible box)?

Not sure if it has any effect. Need more thorough research.

In any case, switching the video to be invisible seems to be a good alternative to a new tracker option.
